ckanext/dcat/harvesters/base.py

```diff
@@ -40,13 +38,24 @@ def munge_tag(tag):
 
 
 class DCATHarvester(HarvesterBase):
-
     MAX_FILE_SIZE = 1024 * 1024 * 50  # 50 Mb
     CHUNK_SIZE = 1024
 
     force_import = False
 
     config = None
+    processors = [
```

Comment:
It might be a good idea to rename `processors` to `config_processors` to be more descriptive.